packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -228,6 +248,34 @@ class CreateCommand extends FlutterCommand {
     }
   }
 
+  _ProjectType getProjectType(Directory projectDir) {
```

Comment:
doesn't really matter but for consistency with the rest of the file, this can likely be private?

Reply:
Done.